open_schools_platform/organization_management/employees/filters.py

```diff
@@ -9,10 +9,12 @@ class EmployeeFilter(BaseFilterSet):
     name = CharFilter(field_name="name", lookup_expr="icontains")
     position = CharFilter(field_name="position", lookup_expr="icontains")
     phone = CharFilter(field_name="employee_profile__user__phone", lookup_expr="icontains")
+    organization_id = CharFilter(field_name="organization__id", lookup_expr="icontains")
+    user_id = CharFilter(field_name="employee_profile__user__id", lookup_expr="icontains")
```

Comment:
it is not necessary to define user_id and organization_id 
you can do it with "fields" list in Meta class, I believe in you

Reply:
somewhere in handlers there can be some mistakes that is realated with this filter
fix VIEWS and SERVICES code too